openrtb_ext/imp_tpmn.go

```diff
@@ -0,0 +1,6 @@
+package openrtb_ext
+
+// ExtImpTpmn defines adman specifiec param
+type ExtImpTpmn struct {
+	inventoryId int `json:"inventoryId"`
```

Comment:
`ExtImpTpmn` will be used with `unmarshal` in adapter code. Note that `unmarshal` works with  public members of struct

```
type ExtImpTpmn struct {
	InventoryId int `json:"inventoryId"`
}
```

Reply:
Thank you for your kind explanation.

update commit : 
[3c38636](https://github.com/prebid/prebid-server/pull/2947/commits/3c38636a36aee5a3fe0827001cfcd40c4fa96416)